src/components/describe/describeWebview.ts

```diff
@@ -0,0 +1,91 @@
+import * as vscode from 'vscode';
+
+export class DescribePanel {
+	public static readonly viewType = 'vscodeKubernetesDescribe';
+	public static currentPanels = [];
+
+	private readonly panel: vscode.WebviewPanel;
+    private readonly extensionPath: string;
+    private content: string;
+	private resource: string;
+
+	private disposables: vscode.Disposable[] = [];
+
+	public static createOrShow(extensionPath: string, content: string, resource: string) {
+		const column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+
+		// If we already have a panel, show it.
+		const currentPanel = DescribePanel.currentPanels[resource];
+		if (currentPanel) {
+			currentPanel.setInfo(content, resource);
+			currentPanel.update();
+			currentPanel.panel.reveal(column);
+			return;
+		}
+		const panel = vscode.window.createWebviewPanel(DescribePanel.viewType, "Kubernetes Describe", column || vscode.ViewColumn.One, {
```

Comment:
Title should say what is being described.

Reply:
Oh you handle this in update

Reply:
not addressesing since not an issue.